src/python_testing/TC_CADMIN_1_3.py

```diff
@@ -0,0 +1,249 @@
+#
+#    Copyright (c) 2024 Project CHIP Authors
+#    All rights reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+# === BEGIN CI TEST ARGUMENTS ===
+# test-runner-runs: run1
```

Comment:
Could you use YAML format for CI args?
See this for reference: https://github.com/project-chip/connectedhomeip/blob/master/src/python_testing/TC_ACE_1_2.py

I'm planing to remove support for not-YAML format in a future, since it does not support line-breaks.

Reply:
Thank you for letting me know about this @arkq, 
I have updated the CI arguments to using YAML format in this test script now. 
I will start doing this for the other CADMIN tests that are currently being worked on.